src/plotrecipes/distplot1D.jl

```diff
@@ -10,7 +10,6 @@
   var   = dp.args[2]
 
   # fit spatial statistics
-  h = normalize(EmpiricalHistogram(sdata, var))
```

Comment:
please heep this line here, and just refer to the `h` variable in the `@series begin h end`